src/playground/sample_data_ingestion/utils/sample_data_generator/sample_data_generator.py

```diff
@@ -0,0 +1,192 @@
+# Faker==13.13.0
+from faker import Faker
```

Comment:
is it a 3rd party library? if so, do we have requirement file to manage the dependencies

Reply:
Yes, I just added the requirements.txt